crates/core-subsystem/core-resolver/src/validation/arguments_validator.rs

```diff
@@ -423,13 +423,14 @@ impl<'a> ArgumentValidator<'a> {
 }
 
 fn get_schema_directives(schema: &Schema, base_type_name: &str) -> Vec<ConstDirective> {
+    let placeholder: Vec<Positioned<FieldDefinition>> = vec![];
     schema
         .type_definitions
         .iter()
         .filter(|td| matches!(&td.kind, TypeKind::Object(_)))
         .flat_map(|td| {
             td.fields()
-                .unwrap()
+                .unwrap_or(&placeholder)
```

Comment:
nit: you can use https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or_default here to simplify it further

Reply:
I think I can't unfortunately:
```
the trait bound `&Vec<Positioned<async_graphql_parser::types::FieldDefinition>>: Default` is not satisfied: the trait `Default` is not implemented for `&Vec<Positioned<async_graphql_parser::types::FieldDefinition>>` 
```

Reply:
I see!